packages/service-client/odsp-client/src/OdspAudience.ts

```diff
@@ -0,0 +1,26 @@
+/*!
```

Comment:
Note: our other modules use `camelCase` file naming, rather than `PascalCase` naming. We should be consistent with this.